src/System.AppContext/tests/AppContext.BaseDirectory.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Diagnostics;
+using Xunit;
+
+namespace System.Tests
+{
+    public partial class AppContextTests
+    {
+        [Fact]
+        [ActiveIssue(6677, PlatformID.OSX)]
+        public void DefaultMatchesGetModuleFileName()
+        {
+            Assert.Equal(GetMainModuleDirectory(), AppContext.BaseDirectory.TrimEnd(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar));
+        }
+
+        private static string GetMainModuleDirectory()
```

Comment:
Hm, that's likely the reason why these tests were left out in the original port. This is the same test from ToF (or at least it was in https://github.com/dotnet/corefx/pull/7310/commits/88b297b8637eec76fa33c00c0d8ee38be0ea1430), so it will need to be fixed or completely removed either way.